airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json

```diff
@@ -126,12 +126,48 @@
       }
     }
   },
-  "authSpecification": {
-    "auth_type": "oauth2.0",
-    "oauth2Specification": {
-      "rootObject": ["authorization", 0],
-      "oauthFlowInitParameters": [["client_id"], ["client_secret"]],
-      "oauthFlowOutputParameters": [["access_token"], ["refresh_token"]]
+  "advanced_auth": {
```

Comment:
do we automatically convert the config object behind the scene or is this a breaking change?

Reply:
@girarda this is neither a breaking change, nor the config object has to be changed. This is just another way to tell the server how to treat the config while going through the oauth flow